roseau/load_flow/network.py

```diff
@@ -324,34 +326,44 @@ def solve_load_flow(
             params=params,
             json=network_data,
             auth=auth,
-            headers={"accept": "application/json"},
+            headers={"accept": "application/json", "rlf-version": __version__},
         )
 
         # Read the response
+        # Check the resonse headers
+        remote_rlf_version = response.headers.get("rlf-new-version")
+        if remote_rlf_version is not None:
+            warnings.warn(
+                message=f"A new version ({remote_rlf_version}) of the library roseau-load-flow is available. Please "
+                f"visit https://github.com/RoseauTechnologies/Roseau_Load_Flow for more information.",
```

Comment:
I like the warning, it will be very helpful for the users to see this warning directly when they start experiencing errors due to version differences.
I think it may be better to link to the documentation instead of the repository. I will write a note to remind myself to write a section about this warning and what the user should do in the docs. For now a link to the installation page will be fine.